scripts/scr_squad_names/scr_squad_names.gml

```diff
@@ -0,0 +1,5 @@
+// Script assets have changed for v2.3.0 see
+// https://help.yoyogames.com/hc/en-us/articles/360005277377 for more information
+function scr_squad_names(){
+
+}
```

Comment:
is this WIP?

Reply:
ah yeah forgot i even made this will delete from this pr

Reply:
ah yeah forgot i even made this will delete from this pr